app/search/styles.scss

```diff
@@ -86,3 +94,77 @@
     display: flex;
     justify-content: space-between;
 }
+
+.search-help-tutorial-1,
+.search-help-tutorial-2,
+.search-help-tutorial-3 {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    background-color: $osf-dark-blue-navbar;
+    color: $color-text-white;
+    width: 400px;
+    margin-left: -0.2rem;
+
+    h2 {
+        font-weight: 400;
+        height: 22px;
+        margin-top: 0.5rem;
+        margin-left: 0.2rem;
+    }
+
+    aside > p {
+        height: 50px;
+        margin-top: 1rem;
+        padding-top: 0.2rem;
+        padding-bottom: 3rem;
+    }
+
+    p {
+        margin-left: 0.2rem;
+    }
+
+    button {
+        width: 65px;
+        color: $color-text-white;
+        border-radius: 2px;
+        overflow: visible;
+    }
```

Comment:
@futa-ikeda we have removed type from the buttons; which lines would you like removed from styles?

Reply:
I ask because with the button class styles removed on lines 128-133 with the requested button types removed it will render as:

![image](https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/80114251-9382-4505-b713-ea066a61ae3a)

After the previous request, the only buttons to have type are in the heading wrapper. I can add types back in, such as primary and secondary, but which would you like?

Reply:
If you implement the chages suggested in [this comment](https://github.com/CenterForOpenScience/ember-osf-web/pull/1877#discussion_r1237138139) and some of the other styling related suggestions I posted, the styles should match what is in the mockups

Reply:
Got it, this has now been updated.